zkevm-circuits/src/evm_circuit/execution/error_return_data_oo_bound.rs

```diff
@@ -141,24 +149,27 @@ impl<F: Field> ExecutionGadget<F> for ErrorReturnDataOutOfBoundGadget<F> {
             ),
         )?;
 
-        // when u64::MAX < data_offset = true, not within u64 range.
-        let data_offset_overflow = U256::from(u64::MAX) < data_offset;
-        self.is_data_offset_within_range.assign(
-            region,
-            offset,
-            F::from(data_offset_overflow as u64),
-        )?;
-        // check `end` if u64 overflow.
-        let end_overflow = U256::from(u64::MAX) < end;
+        let data_offset_overflow = data_offset.to_le_bytes()[N_BYTES_U64..]
+            .iter()
+            .fold(0, |acc, val| acc + u64::from(*val));
+        self.is_data_offset_within_u64
+            .assign(region, offset, F::from(data_offset_overflow))?;
 
-        self.is_end_within_range
-            .assign(region, offset, F::from(end_overflow as u64))?;
+        let remainder_end_overflow = remainder_end.to_le_bytes()[N_BYTES_U64..]
+            .iter()
+            .fold(0, |acc, val| acc + u64::from(*val));
+        self.is_remainder_end_within_u64
+            .assign(region, offset, F::from(remainder_end_overflow))?;
 
         // check if it exceeds last callee return data length
-        let end_u64 = end.low_u64();
+        let reaminder_end_u64 = remainder_end.low_u64();
```

Comment:
there is typo 😃

Reply:
Sorry for that. Fix it in [eb9a1c0](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1321/commits/eb9a1c0ba6d6de30caf72c613ff90753cd1ee1f3).

Reply:
no worry, all good now and tks!